ci/go-publisher-ossf-slsa3.yml

```diff
@@ -0,0 +1,22 @@
+name: SLSA Go releaser
```

Comment:
Please add a short summary:
1. What does this starter workflow light up for customer?
2. Any links for "more details", "support"

Reply:
Done. PTAL

Reply:
I don't see this updated in the PR. Did you may be miss to push the change?

Reply:
done, PTAL